docs/http.md

```diff
@@ -236,7 +236,7 @@ In addition to the [general options](#general-options) the following options are
 |------------|--------------------------------------------------|---------------------------------------------|
 |sources     |`{index};{index}[;{index} ...]` or `all` (default)|Use location with given index as source.     |
 |destinations|`{index};{index}[;{index} ...]` or `all` (default)|Use location with given index as destination.|
-|annotations |`duration` (default), `distance`, or `duration,distance`|Return additional table with distances to the response. Whether requested or not, the duration table is always returned.|
+|annotations |`duration` (default), `distance`, or `duration,distance`|Return additional table with distances to the response. Whether requested or not, the duration table is always returned. Note that `distance` is only currently implemented for CH. Requesting it with MLD algorithm flag will return a `NotImplemented` error.|
```

Comment:
`with MLD algorithm flag`?

is that English?
`with the MLD algorithm flag`?
`with the MLD algorithm`?
`with MLD`?